pkg/api/config.go

```diff
@@ -6,13 +6,22 @@ import (
 
 // Validate validates config
 func (config *ReleaseBuildConfiguration) Validate() error {
+	buildRootImage := config.InputConfiguration.BuildRootImage
+	testBaseImage := config.InputConfiguration.TestBaseImage
 	if config.Tests != nil {
 		for _, test := range config.Tests {
 			if test.As == "images" {
 				return fmt.Errorf("Test should not be called 'images' because it gets confused with '[images]' target")
 			}
 		}
+	} else if buildRootImage != nil && testBaseImage == nil {
+		if buildRootImage.GitSourceReference != nil && buildRootImage.ImageStreamTagReference != nil {
+			return fmt.Errorf("You can't use both git_source_image and image_stream_tag references")
```

Comment:
If you have no formatting directives, use `errors.New()`, not `fmt.Errorf()`

Reply:
Add some more context -- `"both git_source_image and `image_stream_tag` cannot be set for the built_root"`